examples/tcs34725_simpletest.py

```diff
@@ -14,15 +14,9 @@
 
 # Main loop reading color and printing it every second.
 while True:
-    # Read the color as RGB bytes (0-255 values).
-    r, g, b = sensor.color_rgb_bytes
-    print('Detected color: #{0:02X}{1:02X}{2:02X}'.format(r, g, b))
```

Comment:
This seems weird to remove given this is a color temperature. How would I know if something is red?

Reply:
RGB is definitely an entirely different problem with different transformation matrices and issues to address (WHICH RGB working space ... sRGB, Adobe RGB ... which white point, D50, D65, etc.). I can offer some pointers here since I've done a bunch of work in that area, but should definately be a new issue or PR to solve a new problem.

The most 'scientific' method to solve this would be to take the CCT (and unfortunately assume a Duv of 0), convert that to CIE 1931 XYZ tristimulus, and then from that you can accurately convert to any RGB working space. Some of the math is described here, for example: http://www.brucelindbloom.com/index.html?Eqn_RGB_XYZ_Matrix.html

Reply:
As for the RGB algo currently used, I'm going to go out on a limb and says it's a random best guess not rooted in any known conversion formula, and should be replaced for something more accurate like the CCT->XYZ->RGB process mentioned above.

Reply:
OK. I'll put the current code back in so this PR is only DN40 related. And we'll deal with RGB separately.